frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -471,6 +467,28 @@ export function FeatureFlag({ id }: FeatureFlagLogicProps): JSX.Element {
                                         defaultValue={featureFlag.name || ''}
                                     />
                                 </LemonField>
+                                {experiment && (
+                                    <div className="mb-4 p-3 border rounded bg-bg-3000">
+                                        <div className="flex items-center justify-between">
+                                            <div>
+                                                <div className="text-xs text-muted uppercase tracking-wide mb-1">
+                                                    Linked Experiment
+                                                </div>
+                                                <div className="text-sm">
+                                                    This feature flag is powering <strong>{experiment.name}</strong>
+                                                </div>
```

Comment:
this might be too beefy of a visual element? However, I tried another one

<img width="1255" height="369" alt="Image" src="https://github.com/user-attachments/assets/4e94d75d-d4b0-4767-9c80-612840015690" />

and didn't really like that either. Idk. What do folks think?